### PR TITLE
fix(html): new rule for description meta

### PR DESCRIPTION
**Pull request by kptdobe**
Fix https://github.com/adobe/helix-pages/issues/844

### CODE REVIEW

File: src/html.pre.js [hunk 1]
```diff
@@ -81,8 +81,9 @@ async function pre(context, action) {
   let desc = [];
   document.querySelectorAll('div > p').forEach((p) => {
     if (desc.length === 0) {
-      const words = p.textContent.trim().split(/\s+/);
-      if (words.length >= 10) {
+      const text = p.textContent.trim();
+      const words = text.split(/\s+/);
+      if (words.length >= 10 || (words.length === 1 && text.length > 25 && !text.startsWith('http'))) {
```
Comment (rofe) [inline src/html.pre.js:86]: Wouldn't it be safer to (also) check for double-byte characters or if the `charCode` is outside of the latin character range?

**Review by rofe — APPROVED** [review]

File: src/html.pre.js (hunk 1 above)
Comment (kptdobe) [reply src/html.pre.js:86]: In the end, it turns out to be much more complicated than expected: finding the number of words in JP is not easy. And they also have mix characters set which also won't work without more magic: 

```
Adobe XDの強力な機能である「スタック」と「パディング」について佐々木雄平さんが解説します。
```

Only reliable solution I see is to ask them to use the Metadata block.

**Comment by kptdobe** [comment]
I changed the logic to check for at least one word of 25 characters or more. 

```
He caught a pneumonoultramicroscopicsilicovolcanoconiosis.
```

would make it into the description but that would be an edge case. At least in English. In German, we might see more. 
I think the path is anyway to go for the Metadata block.

**Review by rofe — APPROVED** [review]

**Pull request merged**